src/App.js

```diff
@@ -1,21 +1,31 @@
 import Button from './components/Button';
+import getInstance from './utils/audio-manager';
```

Comment:
SInce you've used a default export in `audio-manager.js`, you can import the module with whatever name you want. Maybe it would be worth to use a more explicit name here? The "getInstance" name doesn't really tell you anything about what it actually does or what it is.

Reply:
I just changed it to "AudioManager" .